pkg/controller/scyllacluster/sync_config.go

```diff
@@ -0,0 +1,48 @@
+package scyllacluster
+
+import (
+	"context"
+	"fmt"
+
+	scyllav1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1"
+	"github.com/scylladb/scylla-operator/pkg/controllerhelpers"
+	"github.com/scylladb/scylla-operator/pkg/naming"
+	"github.com/scylladb/scylla-operator/pkg/pointer"
+	"github.com/scylladb/scylla-operator/pkg/resourceapply"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/errors"
+)
+
+func (scc *Controller) syncConfigs(
+	ctx context.Context,
+	sc *scyllav1.ScyllaCluster,
+) ([]metav1.Condition, error) {
+	var errs []error
+	var progressingConditions []metav1.Condition
+
+	cm, err := MakeManagedScyllaDBConfig(sc)
+	if err != nil {
+		return progressingConditions, fmt.Errorf("can't make managed scylladb config: %w", err)
+	}
+
+	cm.SetOwnerReferences([]metav1.OwnerReference{
```

Comment:
why this is outside of `MakeManagedScyllaDBConfig`?

Reply:
modeled after https://github.com/scylladb/scylla-operator/blob/ef4406fd838ed8db3eec6dd87786218ee265b88a/pkg/controller/scylladbmonitoring/sync_grafana.go#L477-L486

the idea behind that is that whoever creates it sets it, so possibly other controllers can use it too. I admit it's not that likely, so I can see this going both ways a bit.

Reply:
I would rather keep it consistent at least across CRD. Having object construction split into multiple places is error prone. 
We can extract it out, once we actually need it.

Reply:
done